File: class08/java/AP0801-OperacoesTexto.java
import java.util.Scanner;

public class OperacoesTexto {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Digite seu nome: ");
        String nome = sc.nextLine();
        System.out.println("Ola, " + nome + "!");
        System.out.println("Seu nome tem " + nome.length() + " caracteres");
        System.out.println("Em maiusculas: " + nome.toUpperCase());
        System.out.println("Em minusculas: " + nome.toLowerCase());
        System.out.println("Primeira letra: " + nome.charAt(0));
    }
}

File: class08/cpp/AP0801-OperacoesTexto.cpp
#include <iostream>
#include <string>
#include <algorithm>
int main() {
    std::string nome;
    std::cout << "Digite seu nome: ";
    std::getline(std::cin, nome);
    std::cout << "Ola, " << nome << "!" << std::endl;
    std::cout << "Seu nome tem " << nome.size() << " caracteres" << std::endl;
    std::string mai = nome, min = nome;
    std::transform(mai.begin(), mai.end(), mai.begin(), ::toupper);
    std::transform(min.begin(), min.end(), min.begin(), ::tolower);
    std::cout << "Em maiusculas: " << mai << std::endl;
    std::cout << "Em minusculas: " << min << std::endl;
    if (!nome.empty()) std::cout << "Primeira letra: " << nome[0] << std::endl;
    return 0;
}
